src/components/map/BiodivInformation.vue

```diff
@@ -32,13 +33,15 @@
     },
 
     computed: {
-      description() {
-        let result = this.data.description || '';
-
-        // very simple html > text interpreter : only html entities
-        result = result.replace(/&eacute;/g, 'é');
-        result = result.replace(/&egrave;/g, 'è');
-        return result;
+      descriptionParagraphs() {
+        // keep line breaks for better readability
+        const paragraphList = (this.data.description || '').split(/<br ?\/?>/);
+        // decode html entities
+        let i = 0;
```

Comment:
a little bit of functional programming would simplify the whole:

```js
return (this.data.description || '')
  .split(/<br ?\/?>/)
  .map(p => utils.decodeHtmlEntities(p);
```

(although I don't remember which es version we support on c2c_ui)